### DISCUSSION TITLE
can't run R after reinstalling

### DISCUSSION THREAD

**Discussion by rezamds** [apache/incubator-mxnet#16459] [Q&A] [answered]
## Description
hi all dear
I removed R from my centos7 and reinstalled it
R is installed but it does not run
## Environment info (Required)

R version 3.6
linux : centos 7

## Error Message:
/usr/lib64/R/bin/exec/R: error while loading shared libraries: libRblas.so: cannot open shared object file: No such file or directory

**Comment by szha** [comment, marked as answer]
This looks like an issue with your R installation. libRblas.so is not a library that mxnet provides.
